### PR TITLE
feat(rating): headless daisyUI ex

### LINKED ISSUE #621: Add rating daisyUI examples
**Description**
Provide usage of rating headless to display a daisyUI rating.

### PR DESCRIPTION
**Pull request by ExFlo**
closes #621

### CODE REVIEW

File: react/demo/src/daisyui/samples/rating/Rating.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	props: Pick<Partial<RatingProps>, 'rating' | 'maxRating' | 'className' | 'onHover' | 'onRatingChange' | 'onLeave' | 'ariaLabel'>,
+) {
+	const [{visibleRating, interactive, stars, className, ariaLabel}, widget] = useWidgetWithConfig(createRating, props, 'rating');
+
+	const starStyle = {
+		cursor: interactive ? 'pointer' : 'default',
+	};
+
+	return (
+		<div className={classNames('join', ['rating', `${className}`])}>
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply react/demo/src/daisyui/samples/rating/Rating.tsx:17]: done

File: react/demo/src/daisyui/samples/pagination/Default.route.tsx [hunk 2]
```diff
@@ -2,8 +2,6 @@ import {Pagination} from './Pagination';
 
 const TailwindDemo = () => (
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply react/demo/src/daisyui/samples/pagination/Default.route.tsx:3]: done

File: demo/src/lib/components-metadata.ts [hunk 3]
```diff
@@ -56,3 +56,19 @@ export const componentsMetadata: {[key: string]: ComponentMetadata} = {
...
+export const daisyUIMetadata: {[key: string]: ComponentMetadata} = {
+	Pagination: {
+		title: 'Pagination',
+		status: 'stable',
+		since: 'v0.1.1',
+	},
+	Rating: {
+		title: 'Rating',
+		status: 'stable',
+		since: 'v0.1.1',
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply demo/src/lib/components-metadata.ts:72]: done

File: demo/src/routes/docs/[framework]/daisyUI/rating/+layout.server.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+export const load = async () => {
+	return {
+		...daisyUIMetadata.Rating,
+		prev: {
+			title: 'Pagination',
+			slug: 'daisyUI/pagination/',
+			subpath: 'headless',
+		},
+		next: {
+			title: 'Component: accordion',
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply demo/src/routes/docs/[framework]/daisyUI/rating/+layout.server.ts:12]: done

File: svelte/demo/src/daisyui/samples/pagination/Default.route.svelte [hunk 5]
```diff
@@ -2,6 +2,4 @@
 	import Pagination from './Pagination.svelte';
 </script>
 
-<h2 class="text-lg mb-2">DaisyUI Example</h2>
-
 <Pagination collectionSize={40} boundaryLinks={true} />
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply svelte/demo/src/daisyui/samples/pagination/Default.route.svelte:5]: done

File: react/demo/src/daisyui/samples/rating/Default.route.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import {useState} from 'react';
+import {Rating} from './Rating';
+
+const TailwindDemo = () => {
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply react/demo/src/daisyui/samples/rating/Default.route.tsx:4]: done

**Review by quentinderoubaix — APPROVED** [review]
thanks !

**Pull request merged**